data/valgrind/valgrind-test.sh

```diff
@@ -22,6 +22,15 @@ function GREP {
     fi
 }
 
+# Check for x86_64 architecture
+# Place this before set -e as it fails on other platforms
+x86_64=0
+uname -i > uname.txt
+grep -i x86_64 uname.txt >/dev/null && x86_64=1
+if [[ $x86_64 == 1 ]]; then
```

Comment:
you could just run simply `if [ $(uname -m) == "x86_64" ] ...`

Reply:
I updated the test script accordingly and updated the test runs to run with the new version